### DISCUSSION TITLE
onnx / tensorflow-onnx and FINN framework

### DISCUSSION THREAD

**Discussion by amroybd** [Xilinx/finn#560] [Q&A] [answered]
Hi, do the FINN framework support the ONNX format converted from the TFLite model using this tf2onnx [converter](https://github.com/onnx/tensorflow-onnx)?
Have a nice day!

**Comment by heborras** [comment, marked as answer]
Hi @amroybd,
currently FINN only supports quantized models, which were trained and exported by [Brevitas](https://github.com/Xilinx/brevitas).
In the future, we are planning to add support for models trained by [QKeras](https://github.com/google/qkeras), in collaboration with our colleagues at [hls4ml](https://fastmachinelearning.org/hls4ml/).
Using the tf2onnx converter as it is currently will likely result in a network, which uses a different quantization to what FINN would expect from Brevitas or QONNX.

**Reply by amroybd** [reply]
I have used hls4ml. It was very interesting. Thank you for your reply
